probe-rs/Cargo.toml

```diff
@@ -164,6 +165,8 @@ crossterm = { version = "<= 0.26.1", optional = true }
 tui = { version = "0.19.0", default-features = false, features = ["crossterm"], optional = true }
 bytesize = { version = "1", optional = true }
 textwrap = { version = "0.16.0", optional = true }
+addr2line = { version = "0.20.0", optional = true }
+strum = { version = "0.24.1", features = ["derive"], optional= true }
```

Comment:
I think this was removed?

Reply:
Oops! I think this crept back in after a rebase, removed it now.